src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -38,6 +40,29 @@ public static Index parseIndex(String oneBasedIndex) throws ParseException {
         return Index.fromOneBased(Integer.parseInt(trimmedIndex));
     }
 
+    /**
+     * Parses two one based index and returns it. Leading and trailing whitespaces will be trimmed.
+     *
+     * @param oneBasedIndex to be parsed.
+     * @return a list of 2 oneBasedIndex.
+     * @throws ParseException if indices are invalid.
+     */
+    public static List<Index> parseTwoIndex(String oneBasedIndex) throws ParseException {
+        String trimmedIndex = oneBasedIndex.trim();
+        String[] indexes = trimmedIndex.split(" ");
+
+        if (indexes.length != 2 || (
+                !StringUtil.isNonZeroUnsignedInteger(indexes[0]) && !StringUtil.isNonZeroUnsignedInteger(indexes[1]))) {
```

Comment:
I think this should be
```suggestion
        if (indexes.length != 2 
                || !StringUtil.isNonZeroUnsignedInteger(indexes[0])
                || !StringUtil.isNonZeroUnsignedInteger(indexes[1])) {
```

Reply:
oops yea